managed-ledger/src/test/java/org/apache/bookkeeper/mledger/util/TestStatsBuckets.java

```diff
@@ -99,4 +99,17 @@ public void test() {
         assertEquals(stats.getCount(), 3);
         assertEquals(stats.getBuckets(), new long[] { 1, 0, 1, 1 });
     }
+
+    @Test
+    public void testAddAll() {
+        StatsBuckets stats = new StatsBuckets(10, 20, 30);
+        stats.addValue(1);
+        stats.addValue(2);
+        stats.refresh();
+        StatsBuckets stats2=new StatsBuckets(10, 20, 30);
```

Comment:
`StatsBuckets stats2 = new StatsBuckets(10, 20, 30);` will be better:)

Reply:
done,thanks